### PR TITLE
Rwahs 175 configure image sizes

### PR DESCRIPTION
**Pull request by kehh**

### CODE REVIEW

**Comment by kehh** [comment]
I have had to override the whole of ca_object_representations in `media_processing.conf`. Look at the individual commits to see what I changed from the default.
18999876cc03f2a418482055a2026b2c37149687 adds the rules
dd2e93dbd483603e489b8cdff795153cabf43413 links rules to versions for each file type
ebe171e9b53689fd3f09c0ee4c776186b1250732 enables downloads of the new media type

**Comment by kehh** [comment]
https://youtrack.gaiaresources.com.au/youtrack/issue/RWAHS-175

**Comment by kehh** [comment]
OK this is now really good to go with human friendly size names

File: conf/app.conf [hunk 1]
```diff
@@ -1,3 +1,12 @@
+
+# -------------------
+# Object representation download options
+# -------------------
+
+# Media versions to provide downloads of
+ca_object_representation_download_versions = [original, "Print-Examination Copy", "View Copy", "View Copy - Multipage", "Thumbnail Copy"]
```
Comment (leftclickben) [inline conf/app.conf]: Should original be uppercase?

Comment (leftclickben) [inline conf/app.conf]: Or make the others lowercase...

**Comment by kehh** [comment]
:fish:

**Comment by leftclickben** [comment]
ca_object_representation_download_versions = ["fish", "squid", "chips", "fried cheese", "scallops"]

**Pull request merged**